extensions/bear/src/search-dropdown.tsx

```diff
@@ -0,0 +1,49 @@
+import { List, Toast, showToast } from "@raycast/api";
+import { useEffect, useState } from "react";
+import { useBearDb } from "./hooks";
+
+type Tag = string;
+type TagsDropdownProps = {
+  onTagChange: (tag: Tag) => void;
+};
+
+const TagsDropdown = (props: TagsDropdownProps) => {
+  const { onTagChange } = props;
+  const [db, error] = useBearDb();
+  const [tags, setTags] = useState<Tag[]>([]);
+
+  useEffect(() => {
+    if (db != null) {
+      setTags(db.getTags());
+    }
+  }, [db]);
+
+  if (error) {
+    showToast(Toast.Style.Failure, "Something went wrong", error.message);
+  }
+  return (
+    <List.Dropdown
+      tooltip="Select a tag"
+      storeValue={true}
+      onChange={(newValue) => {
+        if (newValue === "All tags") {
+          onTagChange("");
```

Comment:
It might be clearer to explicitly use `null` to represent the absence of a tag, rather than an empty string? You'd need to change the type definition to `string | null`.